recipes/liquorice/conda-recipe/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: liquorice
+  version: 0.5.3
+
+source:
+  url: https://github.com/epigen/LIQUORICE/archive/refs/tags/v0.5.3.tar.gz
+  sha256: 08301c9084e00394b68b1cf4d9aeadfb5d6772dfa3c36e7bb8cbe5bc535bf31d
+
+build:
+  number: 0
+  entry_points:
+    - LIQUORICE = liquorice.LIQUORICE:main
+    - LIQUORICE_summary = liquorice.LIQUORICE_summary:main
+  noarch: python
+  script: {{PYTHON}} setup.py install
+
+requirements:
+  host:
+    - python >=3.7
+  run:
+    - python ==3.7.9
```

Comment:
This really need exactly version 3.7.9 and nothing more recent? Similarly, are the exact version constraints below really needed?